charts/script-exporter/Chart.yaml

```diff
@@ -3,4 +3,4 @@ name: script-exporter
 description: Prometheus exporter to execute scripts and collect metrics from the output or the exit status.
 type: application
 version: 0.6.4
-appVersion: v2.14.1
+appVersion: v2.14.2
```

Comment:
Can you revert the version update here please.

Reply:
I admit I don't fully understand the lyfe cycle of this file....

I will revert sure